content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -4,247 +4,206 @@ reviewers:
+  * In this mode, Kubernetes uses the `rbac.authorization.k8s.io` API group to drive authorization decisions, allowing you to dynamically configure permission policies through the Kubernetes API.
 
-```bash
-kubectl auth can-i create deployments --namespace dev
-```
+`Node`
+: A special-purpose authorization mode that grants permissions to kubelets based on the pods they are scheduled to run. To learn more about the Node authorization mode, see [Node Authorization](/docs/reference/access-authn-authz/node/).
 
-The output is similar to this:
+`Webhook`
+: Kubernetes [webhook mode](/docs/reference/access-authn-authz/webhook/) for authorization makes a synchronous HTTP callout, blocking the request until the remote HTTP service responds to the query.You can write your own software to handle the callout, or use solutions from the ecosystem.
 
-```
-yes
-```
+<a id="warning-always-allow" />
 
-```shell
-kubectl auth can-i create deployments --namespace prod
-```
+{{< warning >}}
+Enabling the `AlwaysAllow` mode bypasses authorization; do not use this on a cluster where
+you do not trust **all** potential API clients, including the workloads that you run.
 
-The output is similar to this:
-
-```
-no
-```
-
-Administrators can combine this with [user impersonation](/docs/reference/access-authn-authz/authentication/#user-impersonation)
-to determine what action other users can perform.
-
-```bash
-kubectl auth can-i list secrets --namespace dev --as dave
-```
-
-The output is similar to this:
+You should not use the `AlwaysAllow` mode on a Kubernetes cluster where the API server
+is reachable from the public internet.
+{{< /warning >}}
 
-```
-no
-```
-
-Similarly, to check whether a ServiceAccount named `dev-sa` in Namespace `dev`
-can list Pods in the Namespace `target`:
-
-```bash
-kubectl auth can-i list pods \
-	--namespace target \
-	--as system:serviceaccount:dev:dev-sa
-```
-
-The output is similar to this:
-
-```
-yes
-```
-
-`SelfSubjectAccessReview` is part of the `authorization.k8s.io` API group, which
-exposes the API server authorization to external services. Other resources in
-this group include:
-
-* `SubjectAccessReview` - Access review for any user, not only the current one. Useful for delegating authorization decisions to the API server. For example, the kubelet and extension API servers use this to determine user access to their own APIs.
-* `LocalSubjectAccessReview` - Like `SubjectAccessReview` but restricted to a specific namespace.
-* `SelfSubjectRulesReview` - A review which returns the set of actions a user can perform within a namespace. Useful for users to quickly summarize their own access, or for UIs to hide/show actions.
-
-These APIs can be queried by creating normal Kubernetes resources, where the response "status"
-field of the returned object is the result of the query.
+You can configure the Kubernetes API server's authorizer chain using
+either a command line setting or, as an alpha feature, using a
```

Comment:
```suggestion
either a command line setting or, as a beta feature, using a
```
beta, enabled by default in 1.30

Reply:
We can make that change after v1.30 is released (not before).

Reply:
(done)